.circleci/config.yml

```diff
@@ -869,9 +874,14 @@ jobs:
               exit 1
             fi
       - run:
-          name: Restore Postgres
+          name: Restore Postgres Data
           command: |
             TARGET_ENV=$LOWER_ENV TARGET_ACCOUNT_ID="${LOWER_ENV_ACCOUNT_ID}" ./scripts/postgres/restoreDbFromSource.ts
+      - run:
+          name: Restore Postgres User Permissions
+          command: |
+            cd ./scripts/postgres
+            ./scripts/postgres/create-postgres-users.sh
```

Comment:
```suggestion
            ./create-postgres-users.sh
```

Reply:
This suggestion got lost among my other Github notifications--thank you!!